common/beacon.go

```diff
@@ -11,13 +11,17 @@ const DefaultBeaconID = "default"
 // backward-compatibility reasons
 const DefaultChainHash = "default"
 
-// MultiBeaconFolder
+// MultiBeaconFolder name of the folder where the multi-beacon data is stored
 const MultiBeaconFolder = "multibeacon"
 
 // GetBeaconIDFromEnv read beacon id from an environmental variable.
 // It is used for testing purpose.
 func GetBeaconIDFromEnv() string {
-	return os.Getenv("BEACON_ID")
```

Comment:
if this is really only used for testing purposes, why is it a public method in `Common`?

Reply:
I have no idea!